packages/treasury/bundles/install-on-chain.js

```diff
@@ -1,9 +1,16 @@
 // @ts-check
 import { E } from '@agoric/eventual-send';
 
+import '@agoric/governance/exported.js';
+
 import liquidateBundle from './bundle-liquidateMinimum.js';
 import autoswapBundle from './bundle-multipoolAutoswap.js';
 import stablecoinBundle from './bundle-stablecoinMachine.js';
+import contractGovernorBundle from './bundle-contractGovernor.js';
+import committeeBundle from './bundle-committee.js';
+import binaryVoteCounterBundle from './bundle-binaryVoteCounter.js';
+import { governedParameterTerms } from '../src/params';
```

Comment:
I struggled to find a test where I could walk through this code... it looks like nothing in the tests in this package uses `install-on-chain.js`. But I see the `cosmic-swingset` integration test does.

Reply:
That was how I discovered that I needed to update this was that the integration test failed. It looks like it may be used in bringing up a chain, so I tried to make the settings as realistic as possible.